Tchap/Modules/Favourites/FavouritesCoordinator.swift

```diff
@@ -0,0 +1,76 @@
+// File created from FlowTemplate
+// $ createRootCoordinator.sh Favourites Favourites FavouriteMessages
+/*
+ Copyright 2020 New Vector Ltd
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import UIKit
+
+@objcMembers
+final class FavouritesCoordinator: FavouritesCoordinatorType {
```

Comment:
I'm wondering if we really want this Coordinator. I think `FavouriteMessagesCoordinator` do the job
We don't have currently to open this screen from Objective-C (so we don't need the `FavouritesCoordinatorBridgePresenter` ).
I want to discuss about this with you

Reply:
We kept `FavouritesCoordinatorBridgePresenter` for the future but in this case we use `FavouriteMessagesCoordinator `